### ISSUE TITLE
About training 

### ISSUE THREAD

**Issue by QiqLiang** [Martlgap/octuplet-loss#10]
Hi, could you offer the whole training process code or tell me which project this code is based on ？thank you

**Comment by Martlgap** [comment]
Hi, 
unfortunately, I am not able to provide you any code for the training anymore. I used PyTorch and Tensorflow 2 for training and simply used the provided code as the loss function.
